pkg/controllers/disruption/controller.go

```diff
@@ -184,7 +185,8 @@ func (c *Controller) executeCommand(ctx context.Context, m Method, cmd Command)
 		methodLabel:            m.Type(),
 		consolidationTypeLabel: m.ConsolidationType(),
 	}).Inc()
-	logging.FromContext(ctx).Infof("disrupting via %s %s", m.Type(), cmd)
+	uid := uuid.NewUUID()
```

Comment:
```suggestion
	commandID := uuid.NewUUID()
```

nit: Call it commandID and reference it as that throughout?